packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -1345,6 +1345,19 @@ export class LinuxBuildImage implements IBuildImage {
     });
   }
 
+  /**
+   * Uses a Docker image provided by CodeBuild.
+   *
+   * @returns A Docker image provided by CodeBuild.
+   *
+   * @see https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-available.html
+   *
+   * @param name The image identifier
+   */
+  public static fromCodebuildImage(name: string): IBuildImage {
```

Comment:
Couple of things:

1. Name should be `fromCodeBuildImage` (uppercase "B").
2. Perhaps even `fromCodeBuildImageId` is a better name... What do you think?
3. I think the argument `name` is better called `id`.
4. Can you add `@example 'aws/codebuild/standard:4.0'` to the JS docs?

Reply:
I like all these suggestions--I think they add clarity. Thanks!